enhancements/authentication/pod-security-admission.md

```diff
@@ -111,6 +117,18 @@ spec:
               warn-version: latest
 ```
 
+4. Increase enforce audit profile to "restricted"
```

Comment:
did you mean just `enforce` here?  `audit` became restricted in an earlier step.

Reply:
yes, just `enforce`. As @stlaz mentioned in https://github.com/openshift/enhancements/pull/1132#pullrequestreview-981843217 `warn` and `audit` are really not necessary in this case.

Reply:
reasoning: step 3. (the step we're in right now in the cycle) is a "grace period" where we can introspect out-of-process audit logs (aka `audit`) and in-process client side warnings (aka `warn`) to have a signal what workload needs to be converted. Once we switch to `enforce` admission will hard-reject non-compliant workloads, never reaching `audit` or `warn`.

Reply:
I believe s/enforce audit profile/enforce profile/ is the suggested change which I would also like to see

Reply:
fixed